apps/src/templates/projects/StartNewProject.jsx

```diff
@@ -44,13 +45,37 @@ export default class StartNewProject extends React.Component {
   };
 
   render() {
-    const { canViewAdvancedTools, canViewFullList } = this.props;
+    const { canViewAdvancedTools, canViewFullList, includeDanceParty } = this.props;
     const { showFullList } = this.state;
+    const GAMES_AND_EVENTS =  includeDanceParty ?
+      ['spritelab', 'dance','flappy', 'starwarsblocks', 'starwars', 'bounce', 'sports', 'basketball'] :
+      ['spritelab','flappy', 'starwarsblocks', 'starwars', 'bounce', 'sports', 'basketball'];
+
+    const FOURTH_BASIC_DEFAULT_PROJECT_TYPE =
+      includeDanceParty ? 'dance' : 'flappy';
+
+    const DEFAULT_PROJECT_TYPES_ADVANCED = [
+      'spritelab',
+      'artist',
+      'applab',
+      'gamelab'
+    ];
+
+    const DEFAULT_PROJECT_TYPES_BASIC = [
+      'spritelab',
+      'artist',
+      'minecraft_designer',
+      FOURTH_BASIC_DEFAULT_PROJECT_TYPE
+    ];
+
+    const defaultProjectTypes = canViewAdvancedTools ?
+      DEFAULT_PROJECT_TYPES_ADVANCED : DEFAULT_PROJECT_TYPES_BASIC;
+
     return (
       <div>
         <div style={styles.headingStartNew}>{i18n.projectStartNew()}</div>
         <NewProjectButtons
-          projectTypes={this.props.projectTypes}
+          projectTypes={defaultProjectTypes}
           canViewAdvancedTools={canViewAdvancedTools}
```

Comment:
the `canViewAdvancedTools` prop can be removed from `NewProjectButtons`, right? it looks like that prop is no longer used